JapaneseFilter/sections/specific.txt

```diff
@@ -35,6 +35,9 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+5chan.jp##.linklist > li > a:contains([PR])
```

Comment:
This is not needed, `##.pr` will be stable enough - adding extended syntax is good if non-extended filter doesn' have such a distinguishable selector (e.g. `##.entry-content > p[style]`). Otherwise better not to add extended as it's costy in terms of CPU usage.

Reply:
ok. rule removed.

Reply:
Excues me. This element have to dekete?
`5chan.jp##.linklist > li > a:contains([PR])` This rule is for this element.
<details><summary>screenshot</summary>

![](https://i.uguu.se/pEYnqeYk.png)
</details><br/>

Reply:
Oh, sorry I misunderstood. I thought that is for mobile which is redundant with `##.pr` but you're right, it's needed for PC.

Reply:
Then insert `?` between `##` so that `#?#` and move to css_extended.